edgeless_cli/src/edgeless_cli.rs

```diff
@@ -298,12 +307,16 @@ async fn main() -> anyhow::Result<()> {
                     }
                     log::debug!("Got Config");
                     let conf: CLiConfig = toml::from_str(&std::fs::read_to_string(args.config_file).unwrap()).unwrap();
+                    let function_repository_conf = match conf.function_repository {
+                        Some(conf) => conf,
+                        None => anyhow::bail!("function repository configuration section missing"),
```

Comment:
Would not require that for all subcommands. Especially given it is commented out in the default.

Reply:
Sorry, I don't understand the comment. 

We are in the "Get" branch where the image of a function is downloaded from the function repository.

If the function repository section in the TOML is missing what else can we do other than returning an error? 🤷‍♂️

Reply:
As suggested on slack, I just made a mistake and thought you required that for every command.